src/components/utils/DateFormatter.js

```diff
@@ -16,7 +16,7 @@ import { formatDistance, fromUnixTime } from "date-fns";
 function formatDate(longDate, dateFormat = dateConstants.LONG_DATE_FORMAT) {
     const longDateInt = parseInt(longDate, 10);
     return longDateInt
-        ? lightFormat(toDate(new Date(longDateInt)), dateFormat)
+        ? format(toDate(new Date(longDateInt)), dateFormat)
```

Comment:
The switch from `lightFormat` to `format` caused me a little bit of concern. As far as I can tell `format` is fully compatible with `lightFormat` but not vice versa. If anyone knows of a case (that applies to use) where this is not true, please let me know and I'll create a separate function for formatting ISO8601 dates.

Reply:
I don't see any compatibility issues, but I'm also not sure why we needed to use `format` for the ISO8601 format. The lightFormat docs mention both the `XXX` format token that's being added, and the escaping using single quotes, which appear to be the only differences between LONG_DATE_FORMAT and ISO_8601 in our date constants.

Reply:
I'm not sure either. I couldn't get it to work with `lightFormat`, though.

Reply:
Fair enough! Probably not worth the debugging time to figure out why it didn't work right now, fine to just swap it over I think.